Add Dels officer missions (#12)
1010todd/Galactic-War: data/Dels/Dels missions.txt

		conversation
			`(First Contact: Dels Ufulre) (Under construction)`

# Dels Officer: Missions

mission "Dels Officer: Report to Delro"
	name "Report to <destination>"
	repeat
	landing
	deadline
	description `You have been ordered to go to <destination> for an assignment.`
	destination Delro # maybe have different important systems to report to
	source
		near Vobeelee 1 10
	to offer
		"random" < 30 - "Report to Delro: done" # You get called less frequently over time
		has "GW Dels start: officer"
		0 > "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship"
		not "Dels Officer: Report to Delro: active"
	on offer
		conversation
			`As your ship sinks through the atmosphere of <planet>, your communicator buzzes. The color is orange: answer required.`

> 			label answer

Author: OK, I think I just left it when I removed the choice.

			`	The it reads, "Officer <first>, you are required to go to <destination> by <date>. There you will recieve further orders."`
			choice
				`(Accept)`
					accept
				`"I don't care what consequences you give me, I'm not going!"`
					goto decline
			label decline
				action
					"reputation: Dels Kaote" -= 1

> 				fine 10000

Author: OK, that makes a little more sense.

> 			`Officer <last>, you have dissapinted us. The Kaote Navy will not tolerate such a responce, and as such you will be fined 10,000 credits.`

Author: I'll fix that.

#			`	You prepare to launch.`
#				accept
#	on abort
#		"reputation: Dels Kaote" -= 1
#		fine 10000
#		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
#
#	npc evade
#		government "Dels Ufulre"
#		personality
#			confusion 30
#			plunders harvests target # no staying so they can flee
#		fleet "Dels Ufulre Raid" 2
#		system destination
#
#	npc
#		government "Dels Kaote"
#		personaility
#			confusion 10
#			heroic opportunistic frugal staying
#		fleet "Dels Kaote"
#		system destination
#	on complete
#		payment 100000
#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Patrol Systems"
	landing
	repeat
	priority
	name `Partrol systems`
	source Delro
	stopover
		government "Dels Kaote" Dels
		near Alkurud 0 2
	stopover
		government "Dels Kaote" Dels
		near Alkurud 0 2
	stopover
		government "Dels Kaote" Dels

> 		near Alkurud 0 2

Reviewer: Waypoint would probably be better. Stopover means you have to land on the planet which might get tedious after a while.

Author: The reason I did stopovers was so I could use the planet filter specifications, and I don't know if they work with waypoints.

Author: Never mind, waypoints work with the specifications.

	description `You must patrol <stopovers> to check for Ufulre activity.`
	to offer

> 		has "dels officer: report mission available"

Reviewer: There need to be a way to prevent patrol and escort mission from offering at the same time, maybe have the "Report to Delro" mission predetermine the mission you'll get with branch and random that go to action assigning some condition in the offering conversation.

Author: Would it also work if they became minor, or had priority removed?

Author: I added another offer condition set by randomness.

	on offer
		clear "dels officer: report mission available"
		conversation

> 			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <last>, you are requested for patrol on <stopovers> to check on Ufulre activity."`

Reviewer: The commander probably should just say the mission. Maybe they also hand you data chip to log system activity during patrol.

Author: Sorry, but I'm not sure what you mean by that.

Reviewer: Instead of handing the data pad with instruction maybe the commander should just say what you need to do and hand you data chip to log the patrol instead.

Author: Is `Once you have landed, a Dels commander comes up and says, "Officer <first>, you have been ordered to patrol <waypoints> to check on Ufulre activity."` good?

		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
	on complete
		payment 50000
		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Escort Convoy"
	landing
	repeat
	priority
	name `Escort convoy`
	source Delro
	destination
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
	to offer
		has "dels officer: report mission available"
	on offer
		clear "dels officer: report mission available"
		conversation
			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <last>, you must escort a convoy to <destination> in order to assist those living there."`
			`	You prepare to launch.`
				accept
	on abort
		"reputation: Dels Kaote" -= 1
		fine 10000
		dialog `Officer <last>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`

	npc accompany save
		government "Dels Kaote"
		personaility
			confusion 10

> 			heroic opportunistic frugal staying

Reviewer: They won't follow you if they have staying. Also remove heroic so they aren't suicidal.

		fleet
 			name "Dels civillian"

>  			cargo 50

Reviewer: `cargo` is number of type of commodity they will carry, there's far less than 50 commodity type exist.

Author: I made it quite smaller.

		conversation
			`(First Contact: Dels Ufulre) (Under construction)`

# Dels Officer: Missions

mission "Dels Officer: Report to Delro"
	name "Report to <destination>"
	repeat
	landing
	deadline
	description `You have been ordered to go to <destination> for an assignment.`
	destination Delro # maybe have different important systems to report to
	source
		near Vobeelee 1 10
	to offer
		"random" < 30 - "Report to Delro: done" # You get called less frequently over time
		has "GW Dels start: officer"

> 		#0 > "ships: Interceptor" + "ships: Light Warship" + "ships: Medium Warship" + "ships: Heavy Warship" # doesn't work

Author: OK, that makes sense.

		not "Dels Officer: Report to Delro: active"
	on offer
		conversation
			`As your ship sinks through the atmosphere of <planet>, your communicator buzzes.`

> 			`	The message reads, "Officer <first>, you are required to go to <destination> by <date>. There you will recieve further orders."`

Author: Would this be good?
`choice`
`        "(Open it)"`
`    "The message reads..."`

			choice
				`(Accept)`
					accept
				`"I don't care what consequences you give me, I'm not going!"`
					goto decline
			label decline
			action
				"dels officer: bad officer" ++
	on fail
		"dels officer: bad officer" ++
	on abort
		"dels officer: bad officer" ++
	on complete
		"dels officer: report mission available" = 1
		"Dels Officer: report complete" += 1
		#Cap the factor to 25 to avoid mission not offering again if done for 30 times.
		#So minimal chance of offering is 5%
		"Dels Officer: report complete" <?= 25
		conversation # so you can use branches
			branch escort
				random < 20
			label patrol
			action
				"Dels Officer: Patrol Mission" ++
			goto end
			label escort
			action
				"Dels Officer: Escort Mission" ++

> 			label end

Author: Thanks, I didn't even think of that!

#			`	You prepare to launch.`
#				accept
#	on abort
#		"reputation: Dels Kaote" -= 1
#		fine 10000
#		dialog `Officer <first>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
#
#	npc evade
#		government "Dels Ufulre"
#		personality
#			confusion 30
#			plunders harvests target # no staying so they can flee
#		fleet "Dels Ufulre Raid" 2
#		system destination
#
#	npc
#		government "Dels Kaote"
#		personaility
#			confusion 10
#			heroic opportunistic frugal staying
#		fleet "Dels Kaote"
#		system destination
#	on complete
#		payment 100000
#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Patrol Systems"
	landing
	repeat
	priority
	name `Patrol systems`
	source Delro
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2

> 	description `You must patrol <stopovers> to check for Ufulre activity.`

Reviewer: ```suggestion
	description `You must patrol <stopovers> to check for Ufulre activity then return to <origin> to report.`

```

	to offer
		has "dels officer: report mission available"
		has "Dels Officer: Patrol Mission" # In testing, I had a stack of these but it only offered once.
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Patrol Mission"
		conversation
			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <first>, you must patrol <waypoints> to check on Ufulre activity."`

> 			`	You prepare to launch.`

Reviewer: ```suggestion
			`	You get back to your ship and prepare for the mission.`

```
This flows better I think.

				accept
	on abort
		"dels officer: bad officer" ++
	on complete
		payment 50000
		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Escort Convoy"
	landing
	repeat
	priority
	name `Escort convoy`
	source Delro
	destination
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
	to offer
		has "dels officer: report mission available"
		has "Dels Officer: Escort Mission"
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Escort Mission"
		conversation

> 			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <first>, you must escort a convoy to <destination> in order to assist those living there."`

Author: Thanks, I didn't think of that.

			`	You prepare to launch.`
				accept
	on abort
		"dels officer: bad officer" ++

> 

Reviewer: Since we have `to spawn` now I think maybe there could be occasional ambushing npcs using `random < <somenumber>` node under it for some chance of encountering some resistance.

	npc accompany save
		government "Dels Kaote"
		personaility
			confusion 10
			opportunistic frugal escort
		fleet

>  			name "Dels civillian" # I saw no names

Author: So that's what my error was!

 			cargo 3
 			variant
 				"Dels Manoon" 3
	on complete
		payment 80000

> 		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

Author: I think I'll change that to `You receive a message reading...`

		conversation
			`(First Contact: Dels Ufulre) (Under construction)`

# Dels Officer: Missions

mission "Dels Officer: Report to Delro"
	name "Report to <destination>"
	repeat
	landing
	deadline
	description `You have been ordered to go to <destination> for an assignment.`
	destination Delro # maybe have different important systems to report to
	source
		near Vobeelee 1 10
	to offer
		"random" < 30 - "Dels Officer: report complete"  # You get called less frequently over time

> 

Reviewer: ```suggestion

```

#		system destination
#
#	npc
#		government "Dels Kaote"
#		personaility
#			confusion 10
#			heroic opportunistic frugal staying
#		fleet "Dels Kaote"
#		system destination
#	on complete
#		payment 100000
#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Patrol Systems"
	landing
	repeat
	priority
	name `Patrol systems`
	source Delro
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `You must patrol <stopovers> to check for Ufulre activity then return to <origin> to report.`

	to offer
		has "dels officer: report mission available"
		has "Dels Officer: Patrol Mission" # In testing, I had a stack of these but it only offered once.
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Patrol Mission"
		conversation
			`One you have landed, a Dels commander comes up and says, "Officer <first>, you must patrol <waypoints> to check on Ufulre activity, then return to <origin>."`

> 			`	You get back to your ship and prepare for the mission.`
> 
> 				accept

Reviewer: ```suggestion
			`	You get back to your ship and prepare for the mission.`
				accept

```

mission "Dels Officer: Escort Convoy"
	landing
	repeat
	priority
	name `Escort convoy`
	source Delro
	destination
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
	to offer
		has "dels officer: report mission available"
		has "Dels Officer: Escort Mission"
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Escort Mission"
		conversation
			`Once you have landed, a Dels commander comes up to you and says, "Officer <first>, you must escort a small convoy of three Manoons to <destination> in order to help the people there recover from a recent raid. The ships are in the military priority docking."`
			`	You get back to your ship and prepare for the mission.`
				accept
	on abort
		"dels officer: bad officer" ++

	npc
		government "Dels Ufulre"
		to spawn
			"random" < 50
		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system
			near Alkurud 0 1

	npc
		government "Dels Ufulre"

> 		to spawn
> 			"random" < 20

Reviewer: ```suggestion
		to spawn
			"random" < 20
			"armament deterrence" > 12

```
Since these fleets have chance to spawn Dacar heavy warship maybe lock one of the spawn behind player's fleet strength first so they have less chance of dealing with 2 Dacar at the same time.
Ideally I probably should make the Ufulre fleet more granular and balance out the strength of each variant but that'll be later.

		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system
			near Alkurud 0 1

	npc
		government "Dels Ufulre"
		to spawn
			"random" < 40
		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system destination

	npc accompany save
		government "Dels Kaote"
		personaility
			confusion 10
			opportunistic frugal escort
		fleet
 			names "Dels civillian"

> 

Reviewer: ```suggestion

```

		conversation
			`(First Contact: Dels Ufulre) (Under construction)`

# Dels Officer: Missions

mission "Dels Officer: Report to Delro"
	name "Report to <destination>"
	repeat
	landing
	deadline
	description `You have been ordered to go to <destination> for an assignment.`
	destination Delro # maybe have different important systems to report to
	source
		near Vobeelee 1 10
	to offer
		"random" < 30 - "Dels Officer: report complete"  # You get called less frequently over time
		has "GW Dels start: officer"
		or
			"ships: Interceptor" > 0
			"ships: Light Warship" > 0
			"ships: Medium Warship" > 0
			"ships: Heavy Warship" > 0
		not "Dels Officer: Report to Delro: active"
	on offer
		conversation
			`As your ship sinks through the atmosphere of <planet>, your communicator buzzes notifying you there's a message from the navy command. You open it.`
			`	The message reads, "Officer <first>, you are required to go to <destination> by <date>. There you will receive further orders."`

			choice
				`	(Accept)`
					goto accept
				`	"I don't care what consequences you give me, I'm not going!"`
					goto decline

			label decline
			action
				"dels officer: bad officer" ++

> 			decline

Reviewer: ```suggestion
			``
				decline

```
Fix this too.

		conversation
			`(First Contact: Dels Ufulre) (Under construction)`

# Dels Officer: Missions

mission "Dels Officer: Report to Delro"
	name "Report to <destination>"
	repeat
	landing
	deadline

> 	description `You have been ordered to go to <destination> for an assignment.`

Reviewer: ```suggestion
	description `You have been ordered to go to <destination> for an assignment. You must reach <planet> by <date>.`

```

	destination Delro # maybe have different important systems to report to
	source
		near Vobeelee 1 10

> 	to offer
> 		"random" < 30 - "Dels Officer: report complete"  # You get called less frequently over time
> 		has "GW Dels start: officer"
> 		or
> 			"ships: Interceptor" > 0
> 			"ships: Light Warship" > 0
> 			"ships: Medium Warship" > 0
> 			"ships: Heavy Warship" > 0
> 		not "Dels Officer: Report to Delro: active"

Reviewer: ```suggestion
	to offer
		"random" < 30 - "Dels Officer: report complete"  # You get called less frequently over time
		has "GW Dels start: officer"
		or
			"ships: Interceptor" > 0
			"ships: Light Warship" > 0
			"ships: Medium Warship" > 0
			"ships: Heavy Warship" > 0
		not "Dels Officer: Report to Delro: active"
		not "Dels Officer: Patrol Systems: active"
		not "Dels Officer: Escort Convoy: active"

```
Probably shouldn't offer while you're doing the job.

	on offer
		conversation
			`As your ship sinks through the atmosphere of <planet>, your communicator buzzes notifying you there's a message from the navy command. You open it.`
			`	The message reads, "Officer <first>, you are required to go to <destination> by <date>. There you will receive further orders."`

			choice
				`	(Accept)`
					goto accept
				`	"I don't care what consequences you give me, I'm not going!"`
					goto decline

			label decline
			action
				"dels officer: bad officer" ++
			``
				decline

			

> 			label accept
> 			branch escort
> 				random < 50
> 			label patrol
> 			action
> 				"Dels Officer: Patrol Mission" ++
> 			``
> 				accept
> 
> 
> 
> 			label escort
> 			action
> 				"Dels Officer: Escort Mission" ++
> 			``
> 				accept
> 

Reviewer: ```suggestion
			label accept
			branch escort
				random < 50
			label patrol
			action
				"Dels Officer: Patrol Mission" ++
			``
				goto end

			label escort
			action
				"Dels Officer: Escort Mission" ++
			``
				goto end

			label end
			`	You key in the destination and prepare your ship for the mission.`
				accept
```
Empty ending feels a little odd so maybe just a bit of text.

#	to offer
#		has "dels officer: report mission available"
#	on offer
#		clear "dels officer: report mission available"
#		conversation
#			`One you have landed, a Dels commander comes and gives you a data pad with instructions. It reads, "Officer <first>, you are requested for defense at <destination>. There is an Ufulre attack there."`
#			`	You prepare to launch.`
#				accept
#	on abort
#		"reputation: Dels Kaote" -= 1
#		fine 10000
#		dialog `Officer <first>, the Kaote Navy does not tolerate cowards. You will be fined 10,000 credits for this.`
#
#	npc evade
#		government "Dels Ufulre"
#		personality
#			confusion 30
#			plunders harvests target # no staying so they can flee
#		fleet "Dels Ufulre Raid" 2
#		system destination
#
#	npc
#		government "Dels Kaote"
#		personaility
#			confusion 10
#			heroic opportunistic frugal staying
#		fleet "Dels Kaote"
#		system destination
#	on complete
#		payment 100000
#		dialog `Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission.`

mission "Dels Officer: Patrol Systems"
	landing
	repeat

> 	priority

Reviewer: ```suggestion

```
Apparently this is not necessary.

	name `Patrol systems`
	source Delro
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	waypoint
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `You must patrol <stopovers> to check for Ufulre activity then return to <origin> to report.`

	to offer
		has "dels officer: report mission available"

> 		has "Dels Officer: Patrol Mission" # In testing, I had a stack of these but it only offered once.

Author: I'm starting a conversation here so we can figure out how to get this to offer more.

Author: I have added the `not "Dels Officer: <other__type> Mission"` with this: https://github.com/lumbar527/Galactic-War/commit/41b6f1c2c1c9c44cdbbb348c2e676e6ead6fa586

Author: - [x] Works!

		not "Dels Officer: Escort Mission"
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Patrol Mission"
		conversation
			`One you have landed, a Dels commander comes up and says, "Officer <first>, you must patrol <waypoints> to check on Ufulre activity. After you're done return to <origin>."`

			`	You get back to your ship and prepare for the mission.`
				accept

	on abort
		"dels officer: bad officer" ++
	on complete
		payment 50000
		dialog `You receive a message reading, "Officer <first>, thank you for realizing your role in the Kaote Navy. The commander have authorized <payment> bonus for completing this mission."`

mission "Dels Officer: Escort Convoy"
	landing
	repeat

> 	priority

Reviewer: ```suggestion

```

	name `Escort convoy`
	source Delro
	destination
		government "Dels Kaote" Dels
		near Alkurud 0 2
	description `In order to assist the planets near Ufulre space, you must escort a convoy to <destination>.`
	to offer
		has "dels officer: report mission available"
		has "Dels Officer: Escort Mission"
	on offer
		clear "dels officer: report mission available"
		clear "Dels Officer: Escort Mission"
		not "Dels Officer: Patrol Mission"
		conversation

> 			`Once you have landed, a Dels commander comes up to you and says, "Officer <first>, you must escort a small convoy of three Manoons to <destination> in order to help the people there recover from a recent raid. The ships are in the military priority docking."`

Reviewer: ```suggestion
			`Once you have landed, a Dels commander comes up to you and says, "Officer <first>, you must escort a small convoy of three Manoons to <destination> in order to help the people there recover from a recent raid. The ships are currently parked inside navy hangar 5 and will be ready to launch in an hour."`

```

> 			`	You get back to your ship and prepare for the mission.`

Reviewer: ```suggestion
			`	You went to check the freighters you're escorting and agree on the time to launch then back to your ship and prepare for the mission.`

```

				accept
	on abort
		"dels officer: bad officer" ++

	npc
		government "Dels Ufulre"
		to spawn
			"random" < 50
			"armament deterrence" > 5
		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system
			near Alkurud 0 1

	npc
		government "Dels Ufulre"
		to spawn
			"random" < 20
			"armament deterrence" > 12
		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system
			near Alkurud 0 1

	npc
		government "Dels Ufulre"
		to spawn
			"random" < 40
		personality
			confusion 30
			plunders harvests
		fleet "Dels Ufulre Patrol"
		system destination

	npc accompany save
		government "Dels Kaote"

> 		personaility

Reviewer: ```suggestion
		personality

```
